Sage/carbon
# chore(alert): remove classic theme

removal of classic theme stories from alert component

fixes FE-2900

### Proposed behaviour
Removal of classic theme from component and storybook.

### Current behaviour
Currently, component has classic theme storybook examples.

### Checklist

<del>- [ ] Screenshots are included in the PR </del>
<del>- [ ] Unit tests added or updated </del>
<del>- [ ] Typescript `d.ts` file added or updated </del>

### Additional context
N/A

### Testing instructions
Open storybook.
Check that Alert component has no Classic Theme storybook examples.
